core/src/stores/postgres.rs

```diff
@@ -71,6 +71,92 @@ impl Store for PostgresStore {
         Ok(Project::new_from_id(row_id))
     }
 
+    async fn delete_project(&self, project: &Project) -> Result<()> {
+        let project_id = project.project_id();
+        let pool = self.pool.clone();
+        let mut c = pool.get().await?;
+        let tx = c.transaction().await?;
+
+        // Block executions, Runs & Runs joins
+        let create_proc = r#"
+            CREATE OR REPLACE FUNCTION delete_project_runs(v_project_id BIGINT)
```

Comment:
Let's move to store.rs as CREATE IF NOT EXISTS :pray:

Reply:
(feel free to merge directly after of course!)

Reply:
There is no "create if not exists" for functions. 🙃 

I moved them to store, so now they will be re-created at each deploy. 
I think it's better then managing them with a one time migration because they need to stay up to date at all time and engineers will look at this file to manage any change in the db. I'll add a commit to change it if you're against it 😬

Reply:
Works for me. We just want to make sure concurrent deploys don't crash the thing.